src/redux/actions.js

```diff
@@ -59,10 +60,17 @@ export function clearDistanceFilter() {
   };
 }
 
+export function initializeVisaFilter(visas) {
+  return {
+    type: INITIALIZE_VISA,
+    visas
+  };
+}
+
 export function changeVisaFilter(visa) {
   return {
     type: CHANGE_VISA,
-    visa
+    visa: visa
```

Comment:
Should keep the short hand

Reply:
WHAT THE F*CK, MATT!

Reply:
Not sure why I did that! I'll change it back